Nim/Nim/Program.cs

```diff
@@ -0,0 +1,121 @@
+﻿Console.WriteLine("Welcome to Nim!");
+Console.WriteLine("1 Player(1)");
+Console.WriteLine("2 Players(2)");
+string player = Console.ReadLine();
+int intPlayer = Convert.ToInt32(player);
+
+if (intPlayer == 1)
+{
+    int matches = 25;
+    int loop = 0;
+    Start:
+    if (loop < matches)
+    {
+        Console.Write("|");
+        loop++;
+        goto Start;
+    }
+    Number:
+    loop = 0;
```

Comment:
I really love this approach for printing matches. Without knowing about `for` loops and/or `string` constructors, this is the right way. And the only time I've seen this. Nice!

Reply:
However, in the future, you could also use instead:
`for(int i = 0; i < matches; i++) Console.Write('|');` or even: `Console.Write(new string('|', matches));`